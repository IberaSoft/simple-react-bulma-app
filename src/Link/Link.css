.Link {
    color: white;
    text-align: center;
    padding: 16px;
    text-decoration: none;
    cursor: pointer;
}

.Link.is-active {
    background-color: #333333;
    color: gray;
    cursor: default;
    pointer-events: none;
}