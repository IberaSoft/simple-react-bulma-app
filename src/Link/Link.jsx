import React from 'react';

const Link = (props) => {
    let className = "navbar-item";
    if (props.isActive) className += " is-active";
    return (
        <a className={className} onClick={props.onClick}>{props.children}</a>
    )
}

export default Link;