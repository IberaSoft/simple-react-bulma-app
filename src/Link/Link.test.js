import React from 'react'
import Link from './Link'
import renderer from 'react-test-renderer'; //https://reactjs.org/docs/test-renderer.html

describe('Link component', () => {
    it('Should match the snapshot', () => {
        const wrapper = renderer
            .create(<Link />)
            .toJSON();
        expect(wrapper).toMatchSnapshot();
    });
})