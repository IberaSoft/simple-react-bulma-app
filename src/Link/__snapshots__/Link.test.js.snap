// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component Should match the snapshot 1`] = `
<a
  className="navbar-item "
/>
`;
